explore/src/main/scala/explore/targeteditor/SiderealTargetEditor.scala

```diff
@@ -68,7 +68,8 @@ case class SiderealTargetEditor(
   onClone:           TargetWithId => Callback = _ => Callback.empty,
   renderInTitle:     Option[Tile.RenderInTitle] = None,
   fullScreen:        View[AladinFullScreen],
-  globalPreferences: View[GlobalPreferences]
+  globalPreferences: View[GlobalPreferences],
+  sequenceChanged:   Callback = Callback.empty
```

Comment:
May I suggest changing this parameter name to `invalidateSequence`? It took me a bit to understand what it did.